### PR TITLE
Feature/add dal and bll

### LINKED ISSUE #5: Add DAL and BLL

### PR DESCRIPTION
**Pull request by KurnosovaElena**

### CODE REVIEW

File: Management_System/BLL/DI/ServicesConfguration.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using DAL.DI;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace BLL.DI;
```
Comment (Vlada25) [inline Management_System/BLL/DI/ServicesConfguration.cs:5]: Spacing pls

File: Management_System/DAL/Constants/Connections.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace DAL.Constants;
+
+internal static class Connections
+{
+    internal const string DBConnection = "DBConnection";
+}
```
Comment (5Ka-me) [inline Management_System/DAL/Constants/Connections.cs]: Why this class and field have internal modifier?

File: Management_System/DAL/Entities/UserBoardEntity.cs [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿using Microsoft.EntityFrameworkCore;
 using System.ComponentModel.DataAnnotations;
 
-namespace ManagementSystem.Entities;
+namespace DAL.Entities;
 
 [PrimaryKey(nameof(UserId), additionalPropertyNames: [nameof(BoardId)])]
```
Comment (5Ka-me) [inline Management_System/DAL/Entities/UserBoardEntity.cs]: What is it?

File: Management_System/DAL/Entities/UserEntity.cs [hunk 4]
```diff
@@ -1,4 +1,4 @@
-﻿namespace ManagementSystem.Entities;
+﻿namespace DAL.Entities;
 
 public class UserEntity
 {
```
Comment (Vlada25) [inline Management_System/DAL/Entities/UserEntity.cs:4]: Why can't FirstName, LastName and Email have null value here?

File: Management_System/DAL/Entities/UserBoardEntity.cs (hunk 3 above)
Comment (DmitriyG0r4anin) [inline Management_System/DAL/Entities/UserBoardEntity.cs]: You can configure PK via fluent configuration in OnModelCreating method

File: Management_System/DAL/Entities/UserTaskEntity.cs [hunk 5]
```diff
@@ -1,7 +1,7 @@
-﻿using ManagementSystem.Enums;
+﻿using DAL.Enums;
 using Microsoft.EntityFrameworkCore;
 
-namespace ManagementSystem.Entities;
+namespace DAL.Entities;
 
 [PrimaryKey(nameof(UserId), additionalPropertyNames: [nameof(TaskId)])]
```
Comment (DmitriyG0r4anin) [inline Management_System/DAL/Entities/UserTaskEntity.cs]: same to previous

**Review by DmitriyG0r4anin — APPROVED** [review]

File: Management_System/BLL/DI/ServicesConfguration.cs (hunk 1 above)
Comment (KurnosovaElena) [reply Management_System/BLL/DI/ServicesConfguration.cs:5]: Space between using and namespace

File: Management_System/DAL/Entities/UserBoardEntity.cs (hunk 3 above)
Comment (KurnosovaElena) [reply Management_System/DAL/Entities/UserBoardEntity.cs]: composite primary key

**Review by Vlada25 — APPROVED** [review]

**Pull request merged**